public/index.html

```diff
@@ -4,6 +4,8 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>ID Simulator</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.1/css/bulma.min.css">
```

Comment:
@anthkris Did I accidentally put this back in?

Reply:
@garajo Yup. that shouldn't be necessary any more.

Reply:
ok, removed